### PR TITLE
Don't replace values that are functions with empty object

### PR DESCRIPTION
**Pull request by rupurt**

### CODE REVIEW

**Comment by rupurt** [comment]
@domchristie any chance you can take a look at this?

**Comment by domchristie** [comment]
@rupurt thanks for your contribution. I will review this when I can. Thanks again.

**Comment by rupurt** [comment]
Hey @domchristie,

Is there anything I can do to help this along?

File: humps.js [hunk 1]
```diff
@@ -25,6 +25,9 @@
         output.push(_processKeys(convert, obj[i], options));
       }
     }
+    else if(_isFunction(obj)) {
```
Comment (domchristie) [inline humps.js]: Could this be done as part of the checks on line 14?

Comment (rupurt) [inline humps.js]: Totally. I'll update and force push

**Comment by rupurt** [comment]
@domchristie updated with suggestion

**Comment by domchristie** [comment]
Released in v2.0.0. Thanks again, and apologies for the delay!

**Comment by rupurt** [comment]
No worries. Thanks for merging.

**Pull request merged**